docs/lua-records/functions.rst

```diff
@@ -317,6 +318,18 @@ Reverse DNS functions
     $ dig +short AAAA 2001-a-b--1.static6.example.com @ns1.example.com
     2001:a:b::1
 
+.. function:: filterForward(address, masks[, fallback])
+
+  Used for limiting the output of :func:`createForward` and :func:`createForward6` to a set of netmasks.
+
+  :param address: A string containing an address, usually taken directly from :func:`createForward: or :func:`createForward6`.
+  :param masks: A NetmaskGroup; any address not matching the NMG will be replaced by the fallback address.
+  :param fallback: A string containing the fallback address. Defaults to ``0.0.0.0`` or ``::``.
```

Comment:
I wonder if a different fallback for v6 would make sense?

Reply:
I now also wonder if we should produce a NODATA or NXDomain when no fallback is set.. But I think LUA records can't do that?

Reply:
I did not invent the fallbacks in this PR, to be clear - they were already there for names that could not be parsed into addresses. That said, I'm open to suggestions.

We can generate a malformed NODATA by returning an empty table, but DNSSEC hates it.

Reply:
Right, so perhaps we would better leave it as it is right now?

Reply:
I'd like that for this PR, yes, unless we have an excellent idea for a better address :)

Reply:
> I now also wonder if we should produce a NODATA or NXDomain when no fallback is set.. But I think LUA records can't do that?

There's a CNAME chaining trick for NXDOMAIN, and I -think- `filterForward` could even be used in that, but I'd like to give that more thought outside of this PR.